dev/releases/release_notes.py

```diff
@@ -165,16 +167,33 @@ def get_pr_list(date: str, extra: str) -> List[Dict[str, Any]]:
 def pr_to_md(pr: Dict[str, Any]) -> str:
     """Returns markdown string for the PR entry"""
     k = pr["number"]
-    title = pr["title"]
-    return f"- [#{k}](https://github.com/oscar-system/Oscar.jl/pull/{k}) {title}\n"
+    if has_label(pr, 'release notes: use body'):
+        mdstring = pr['body'].replace("- ", f"- [#{k}](https://github.com/oscar-system/Oscar.jl/pull/{k}) ")
```

Comment:
Won't this also replace any `- ` that occurs in the middle of a description? Might be better to use a regex anchor, i.e., something like
```suggestion
        mdstring = pr['body'].replace(r"^- ", f"- [#{k}](https://github.com/oscar-system/Oscar.jl/pull/{k}) ")
```